### DISCUSSION TITLE
Use "backtick" (`) in parameter names to render in the doc string hover

### DISCUSSION THREAD

**Discussion by diogo-rossi** [haiiliin/abqpy#2956] [Ideas]
VS Code does not have a good doc string render.

To improve that, I found a way to make doc string clearer is to use "backticks" in argument names, so that they got render in VS Code hover make the text more organized.

To sphinx documentation, these backticks are "transparent"

### Current docstring (no backticks):

![image](https://user-images.githubusercontent.com/57204023/197085821-fd8846e6-1a0b-403f-bcbf-9738f1c26858.png)

### Propose docstring (argument names with backticks):

![image](https://user-images.githubusercontent.com/57204023/197085902-1b30f2b3-a7b9-4b5e-9283-c305d4b08f6f.png)

## Necessary trade-off

I found that to the sphinx documentation, this are transparent only if the doc string format already have the "type" of the parameter (like numpy doc string).

If not, sphinx does not put the types automatically.

## OBS
This is a proposal to try to solve a VS Code issue. If it brings more problem than benefits, we should left it to the VS Code render be improved.

**Comment by haiiliin** [comment]
Since it is just a VSCode issue, I prefer not to make any changes to solve the VSCode-specific issue, and it does not seem to be consistent with the NumPy-style docstring.

**Reply by diogo-rossi** [reply]
Ok. It is reasonable.
